### DISCUSSION TITLE
Default mobile difficulty preset.

### DISCUSSION THREAD

**Discussion by o-dasher** [ppy/osu#20149] [Ideas]
So we have osu!lazer, it aims to provide proper cross-platform support, although that, it is still pretty unfair with mobile players to play with the same difficulty settings as pc players by default.

I would like something like what osu!droid does, it lowers both `Circle Size` and `Overall Difficulty` to aim to account for the difficulty that is playing with mobile devices (Increase input lag overall and smaller screens).

My suggestion would be so that the default mobile preset would lower the `Circle Size` by 5 and the `Overall Difficulty` being reduced by 5 the same way.

It could work like a system mod preset,  for this change to be possible without drawbacks such as not being able to select other mod presets, mod presets should be able to be combined accounting for the incompatible mods of each preset.

With that it would also be interesting to provide relative difficulty adjust settings like "-5 or a division from the beatmap base od/cs" so it is easier to implement this, and it would be a great addition too.

**Comment by peppy** [comment]
We will add the ability to load the last preset on game startup in the future.

Also an aside, but "increase input lag" is probably an android thing. iOS is comparable or better than desktop.

**Reply by o-dasher** [reply]
This in my opinion doesn't solve the issue with that because after all it would only work  for players which have already some experience into the game, with an actual mobile preset it would help people switching from osu!droid to lazer and newcomers as a whole.

For the input lag part, i play on a 240hz touch sampling phone with a 120hz screen and it is still not comparable to desktop on terms of input lag so i think it's fair for the overall difficulty be reduced in this default mobile preset.

This could also maybe be a thing on the "New player guide screen", it would say something like "Do you want an easier time while playing, we will adjust the difficulty so it is better fit to an mobile device if so, but you will be rewarded with less performance points (insert link to what is pp)", otherwise keep the pc settings.

**Reply by o-dasher** [reply]
For the text it also could maybe say "Are you coming from osu!droid, we will adjust the difficulty to provide a better mobile experience" something like that, and yes i aknowledge osu!droid isn't affiliated with osu! but still is a reality

**Reply by o-dasher** [reply]
> I think that's what easy mod is.

No, despite the values being different what this mod aims to do is to provide a default difficulty adjust relative to the base adjust so you could stack this with other mods such eas easy itself, it maybe does not need to be a mod preset and just a mod too but i ain't sure on that.

[a comment by an automated account is not shown]

**Comment by o-dasher** [comment]
Thinking about it a mod would be better since it would be more explicit that a play was set in a mobile device and it would require way way less overengineering

**Reply by cdwcgt** [reply]
So how is its difficulty multiplier defined?
and can't enable it on PC (non-moblie device)?
